content/workshops/containers_101/exercise1.2.adoc

```diff
@@ -0,0 +1,272 @@
+---
+title: "Exercise 1.2 - Working with Docker Images"
+workshops: containers_101
+workshop_weight: 40
+layout: lab
+---
+
+:badges:
+:icons: font
+:imagesdir: /workshops/containers_101/images
+:source-highlighter: highlight.js
+:source-language: yaml
+
+= Exercise 1.2 - Working with Docker Images
+
+== Run Docker
+
+When you execute a docker run command, you essentially spin up and create a new container from a Docker image. That container consists of the contents of the image, plus features based on any additional options you pass on the docker run command line.
+
+The command you pass on the docker run command line sees the inside the container as its running environment so, by default, very little can be seen of the host system. For example, by default, the running applications sees:
+
+- The file system provided by the Docker image.
+
+- A new process table from inside the container (no processes from the host can be seen).
+
+- New network interfaces (by default, a separate docker network interface provides a private IP address to each container via DHCP).
+
+If you want to make a directory from the host available to the container, map network ports from the container to the host, limit the amount of memory the container can use, or expand the CPU shares available to the container, you can do those things from the docker run command line.
+
+Launch a container and explore inside interactively.
+
+
+.Docker run command
+[source, bash]
+----
+docker run --rm -it registry.access.redhat.com/rhel7/rhel /bin/bash
+----
+
+Commands to try;
```

Comment:
You have a semi-colon `;` instead of a colon `:` at the end. At least I think that your intention is to have a colon at the end.